diff --git a/modules/comment/comment.module b/modules/comment/comment.module
index 1505523..90f6fc9 100644
--- a/modules/comment/comment.module
+++ b/modules/comment/comment.module
@@ -830,10 +830,13 @@ function comment_links($comment, $return = 1) {
         'title' => t('edit'),
         'href' => "comment/edit/$comment->cid"
       );
-      $links['comment_reply'] = array(
-        'title' => t('reply'),
-        'href' => "comment/reply/$comment->nid/$comment->cid"
-      );
+      $node = node_load($comment->nid);
+      if ($node->type != 'support_ticket') {
+        $links['comment_reply'] = array(
+          'title' => t('reply'),
+          'href' => "comment/reply/$comment->nid/$comment->cid"
+        );
+      }
     }
     else if (user_access('post comments')) {
       if (comment_access('edit', $comment)) {
@@ -842,10 +845,13 @@ function comment_links($comment, $return = 1) {
           'href' => "comment/edit/$comment->cid"
         );
       }
-      $links['comment_reply'] = array(
-        'title' => t('reply'),
-        'href' => "comment/reply/$comment->nid/$comment->cid"
-      );
+      $node = node_load($comment->nid);
+      if ($node->type != 'support_ticket') {
+        $links['comment_reply'] = array(
+          'title' => t('reply'),
+          'href' => "comment/reply/$comment->nid/$comment->cid"
+        );
+      }
     }
     else {
       $node = node_load($comment->nid);
@@ -1048,7 +1054,12 @@ function comment_render($node, $cid = 0) {
     // If enabled, show new comment form if it's not already being displayed.
     $reply = arg(0) == 'comment' && arg(1) == 'reply';
     if (user_access('post comments') && node_comment_mode($nid) == COMMENT_NODE_READ_WRITE && (variable_get('comment_form_location_'. $node->type, COMMENT_FORM_SEPARATE_PAGE) == COMMENT_FORM_BELOW) && !$reply) {
-      $output .= comment_form_box(array('nid' => $nid), t('Post new comment'));
+      if ($node->type == 'support_ticket') {
+        $output .= comment_form_box(array('nid' => $nid), '');
+      }
+      else {
+        $output .= comment_form_box(array('nid' => $nid), t('Post new comment'));
+      }
     }
 
     $output = theme('comment_wrapper', $output, $node);
